src/value.rs

```diff
@@ -142,10 +142,22 @@ impl<'lua> MultiValue<'lua> {
         self.0.push(value);
     }
 
+    pub(crate) fn append(&mut self, values: Self) {
```

Comment:
Minor: as an occasional Python user I assumed append added one value.  Maybe `extend` would be better?

Reply:
In rust the following naming convention is used in stdlib:
- [push](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.push) - single owned value
- [append](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.append) - push all values from `&mut Self`
- [extend](https://doc.rust-lang.org/std/iter/trait.Extend.html#tymethod.extend) - with values returned from `Iterator`

The argument should be a `&mut Self` though.

Reply:
Ok, fair enough!